src/main/resources/static/js/custom/notice-form.js

```diff
@@ -1,4 +1,4 @@
-import { LanguageSelector, NoticeSubtypeSelector, SdkVersionSelector } from "./context.js";
+import { Context, } from "./context.js";
```

Comment:
{ Context, }
Maybe some lesser known browsers could have issues with that trailing comma, I would remove it just to be safe.